src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -100,8 +100,10 @@ public void frameworkMessage(ExecutorDriver driver, byte[] msg) {
     if (messageStr.equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
         || messageStr.equals(HDFSConstants.NAME_NODE_BOOTSTRAP_MESSAGE)) {
       if (nameDir.exists()) {
+        // TODO (elingg) test failure scenario to make sure NN restarts with the
+        // appropriate sequence of events
         log.info(String
-            .format("NameNode data directory %s already exists, not formatting just starting",
+            .format("NameNode data directory %s already exists, not formatting",
```

Comment:
But wouldn't you need to start the NN/ZKFC processes if they aren't already started? Even if the /name directory already exists from a previous run?